Docs/sphinx_documentation/source/Faq.rst

```diff
@@ -0,0 +1,132 @@
+.. role:: cpp(code)
+
+
+Frequently Asked Questions
+==========================
+
+
+**Q.** Why am I getting a segmentation fault after my code runs?
+
+**A.** Do you have :cpp:`amrex::initialize(); {` and :cpp:`} amrex::finalize();`
+at the beginning and end of your code? For all AMReX commands to function
+properly, including to release resources, they need to be contained
+between these two curly braces. In the `Initialize and Finalize`_ section,
+these commands are discussed further detail.
+
```

Comment:
Also, don't forget to pass the input file as the first argument.

Reply:
I think not including the input file gives a seg fault immediately. But please let me know if I'm mistaken.

Reply:
For some codes, yes. For some others, it gives an abort instead.